src/main.ts

```diff
@@ -14,6 +14,10 @@ const root =
     ? path.join(process.env.GITHUB_WORKSPACE, ".") // Running on GH but not tests
     : path.join(__dirname, "..") // default, but should never trigger
 
+const userAgentVersion = process.env.GITHUB_ACTION_REF
+  ? process.env.GITHUB_ACTION_REF
+  : "unknown"
```

Comment:
Would this also work?

```suggestion
const userAgentVersion = process.env.GITHUB_ACTION_REF || "unknown"
```

Reply:
TBH I don't know 😅

Reply:
I will keep the first option `?:`, since it has been used in other parts of the code as well :)